kedro/runner/runner.py

```diff
@@ -17,6 +18,7 @@
 )
 from typing import Any, Collection, Iterable, Iterator
 
+import pandas as pd
```

Comment:
This would mean having to add `pandas` as a dependency and especially after the discussion we had today I don't think that will be a good idea 😅

Reply:
Yeah, that's why it doesn't seem like a good solution. Otherwise, we need to copy objects in all cases.

From https://github.com/scikit-learn/scikit-learn/issues/28781 it seems like a `scikit-learn` problem when multiprocessing. So the current plan is to report our case - just need to create an example without kedro-related code.